kornia/augmentation/augmentation.py

```diff
@@ -387,10 +388,10 @@ def __init__(
             warnings.warn("interpolation is deprecated. Please use resample instead.", category=DeprecationWarning)
             self.resample = Resample.get(interpolation)
         self.resample = Resample.get(resample)
-        self.align_corners = align_corners
+        self.align_corners = _parse_align_corners(align_corners, self.resample)
         self.flags: Dict[str, torch.Tensor] = dict(
             interpolation=torch.tensor(self.resample.value),
-            align_corners=torch.tensor(align_corners)
+            align_corners=torch.tensor(self.align_corners)
```

Comment:
This will fail for `resample=Resample.NEAREST`, since `align_corners` would be `None`. But then, this is not working as of now, since `torch.nn.functional.interpolate` would raise an error with `align_corners=False`.